sqlite.go

```diff
@@ -630,6 +630,7 @@ type rows struct {
 	stmt   *stmt
 	closed bool
 
+	colType  [16]sqliteh.ColumnType
```

Comment:
It would be worth documenting why 16 here too.

Reply:
Done. Documented. And changed it to 23 in the process of documenting, because control has one particular hot query with exactly 16 columns. Too close for comfort.